client/store/index.js

```diff
@@ -0,0 +1,5 @@
+import { createStore } from 'redux';
+import rootReducer from '@client/reducers';
+import middleware from '@client/middleware';
+
+export default createStore(rootReducer, middleware);
```

Comment:
I see. So there is one data store for the whole application. And then you choose what data you want in your stateToProps call?